content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,135 @@
-create a webhook either automatically or manually.
+If your **Webhook Signing Secret** is required to connect with your Stripe account, you can
+create a webhook automatically or manually.
 
 .. tabs::
    .. tab:: Create the webhook automatically
 
       Make sure your :ref:`Publishable and Secret keys <stripe/api_keys>` are filled in, then click
-      on the *Generate your Webhook* button.
+      :guilabel:`Generate your Webhook`.
 
    .. tab:: Create the webhook manually
 
-      Visit the `webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into your
-      Stripe dashboard and go to :menuselection:`Developers --> Webhooks`. Then, click on **Add
-      endpoint** in your **Hosted endpoints** and insert the following data into the pop-up form:
-
-      - | In the **Endpoint URL**, enter your Odoo database's URL followed by
-        | `/payment/stripe/webhook`.
-        | For example: `https://yourcompany.odoo.com/payment/stripe/webhook`
-      - At the end of the form, you can **Select events** to listen to. Click on it and, in the
-        **Checkout** section, select **checkout.session.completed**.
-
-        .. note::
-           It is possible to select other events, but they are currently not processed by Odoo.
-
-      When you click on **Add endpoint**, your Webhook is configured. You can then click on
-      **reveal** to display your signing secret.
+      #. Go to the `Webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into
+         your Stripe dashboard and go to :menuselection:`Developers --> Webhooks`.
+      #. In the :guilabel:`Hosted endpoints` section, click :guilabel:`Add endpoint`. Then, in the
+         :guilabel:`Endpoint URL` field, enter your Odoo database's URL, followed by
+         `/payment/stripe/webhook`, e.g., `https://yourcompany.odoo.com/payment/stripe/webhook`.
+      #.  Click :guilabel:`Select events` at the bottom of the form, then, in the
+          :guilabel:`Checkout` section, select :guilabel:`checkout.session.completed`, then click
+          :guilabel:`Add events`.
+      #. Click :guilabel:`Add endpoint`, then click :guilabel:`Reveal` and save your
+         :guilabel:`Signing secret` for later.
+      #. In Odoo, :ref:`navigate to the payment provider Stripe
+         <payment_providers/supported_providers>`.
+      #. In the :guilabel:`Credentials` tab, fill the :guilabel:`Webhook Signing Secret` field with
+         the value you previously saved.
+
+      .. note::
+         You can select other events, but they are currently not processed by Odoo.
 
 .. _stripe/local-payment-methods:
 
 Enable local payment methods
 ============================
 
-Local payment methods are payment methods that are only available for certain merchants and
-customers countries and currencies.
+Local payment methods are payment methods that are only available for specific merchants and
```

Comment:
```suggestion
Local payment methods are payment methods that are only available for specific merchants' and
```
I'm not positive about this apostrophe - it depends on if local payment methods are only available for specific merchants or for merchants of specific countries and currencies.